app/src/main/java/com/example/sharingang/items/ItemDatabase.kt

```diff
@@ -5,7 +5,7 @@ import androidx.room.RoomDatabase
 import androidx.room.TypeConverters
 import com.example.sharingang.DateConverter
 
-@Database(entities = [Item::class], version = 1)
+@Database(entities = [Item::class], version = 2)
```

Comment:
What is the difference between using version 1 and version 2?

Reply:
So like, when you change the database schema, you're supposed to bump up the version, so it knows how to handle old data.